model/src/cmake/switches.json

```diff
@@ -282,7 +282,7 @@
             },
             {
                 "name": "NL2",
-                "build_files": ["w3snl2md.F90", "mod_xnl4v5.f90", "serv_xnl4v5.f90", "mod_fileio.f90", "mod_constants.f90"],
+                "build_files": ["w3snl2md.F90", "mod_xnl4v5.f90", "serv_xnl4v5.f90", "mod_fileio.f90", "mod_constants.f90", "mod_mpicomm.f90"],
```

Comment:
I do not see a file mod_mpicomm.f90. 

Also I think mpicomm.F90 should be adde with the MPI switch not NL2.

Reply:
I can revert this change, I was initially confused about whether mod files were separately needed